releng/org.eclipse.xsemantics.parent/pom.xml

```diff
@@ -63,7 +63,7 @@ Contributors:
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<xtext.version>2.13.0</xtext.version>
 		<xtextVersion>${xtext.version}</xtextVersion>
-		<mwe2Version>2.9.0.201605261059</mwe2Version>
+		<mwe2Version>2.9.1</mwe2Version>
```

Comment:
You must specify the complete version, I think it is 2.9.1.201705291010 (see the Travis log)